.github/workflows/lint-php.yml

```diff
@@ -25,19 +25,20 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        php-versions: [ "7.4", "8.0", "8.1", "8.2" ]
```

Comment:
**nitpick:** Still runs on `stable*` where 7.4 is supported.

Reply:
Good catch! I added PHP 7.4 back.